src/pages/onboarding/subpages/create-transcription-job.jsx

```diff
@@ -182,16 +183,17 @@ export const CreateTranscriptionJobPage = () => {
                       ]}
                     />
                   </FormField>
```

Comment:
This still applies a row-gap styling when the Box below is not displayed. Not all browsers can apply a :has(... ) solution in the parent, plus there could be a condition where multiple last items are not displayed, but the row-gap is still added.

Reply:
@gethinwebster conditionally rendering this element seems to be the best way to progressively disclose it and preserve the styling and aria labelling. However other examples show the progressive disclosing being done via wrapping with the <Box> component and utilize the display prop.